win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example

```diff
@@ -14,7 +21,13 @@ init_config:
 
 instances:
 
-  -
+    ## @param submit_all_events - boolean - optional - default: false
+    ## By default, all events will not be submitted but rather specified by filters.
+    ## Overrides `submit_all_events` in `init_config`.
+    ##
+    ## WARNING: Enabling this will severely impact performance and clutter your event stream.
+    #
+  - submit_all_events: false
```

Comment:
Why do you have the setting both globally and locally? Can't we have just one of those?

Reply:
The global is a shortcut if we have many instances. I don't have a strong opinion about having it tho.